tests/data_snack/connections/test_memcached.py

```diff
@@ -0,0 +1,54 @@
+from unittest.mock import MagicMock
+
+import pytest
+
+from data_snack.connections.memcached import MemcachedConnection
+
+
+@pytest.fixture
+def connection() -> MemcachedConnection:
+    """A `MemcachedConnection` with mocked connection to the actual database."""
+    return MemcachedConnection(connection=MagicMock())
+
+
+def test_get(connection: MemcachedConnection) -> None:
+    """Testing getting a single value based of the provided key."""
+    connection.connection.get.return_value = "value"
+
+    result = connection.get("key")
+    assert result == "value"
+    connection.connection.get.assert_called_with("key")
+
+
+def test_set(connection: MemcachedConnection) -> None:
+    """Testing setting a single value based for the provided key."""
+    connection.connection.set.return_value = "key"
+
+    result = connection.set("key", "value")
+    assert result == "key"
+    connection.connection.set.assert_called_with("key", "value")
+
+
+def test_get_many(connection: MemcachedConnection) -> None:
+    """Testing getting multiple values based on a provided list of keys."""
+    connection.connection.get_many.return_value = {"key1": "value1", "key2": "value2"}
+
+    result = connection.get_many(["key1", "key2"])
+    assert result == {"key1": "value1", "key2": "value2"}
```

Comment:
We could skip this type of asserts since it is just a mock check

Reply:
same for other cases

Reply:
Are you sure? We are mocking function called inside, so we are checking if the function is calling `self.connection.get_many` correctly